test/tstQueryTreeManufacturedSolution.cpp

```diff
@@ -219,7 +219,7 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(structured_grid, TreeTypeTraits,
   std::uniform_int_distribution<> dist_x(0, nx - 1);
   std::uniform_int_distribution<> dist_y(0, ny - 1);
   std::uniform_int_distribution<> dist_z(0, nz - 1);
-  std::uniform_real_distribution<double> dist_shift(-0.45, 0.45);
+  std::uniform_real_distribution<float> dist_shift(-0.45f, 0.45f);
```

Comment:
Does CTAD not work?  Do you need Bothe the explicit template parameter and the suffix?

Reply:
#850.